crates/bevy_pbr/src/lib.rs

```diff
@@ -79,6 +79,8 @@ pub mod graph {
         /// Label for the screen space ambient occlusion render node.
         ScreenSpaceAmbientOcclusion,
         DeferredLightingPass,
+        /// Label for the compute shader mesh uniforms building pass.
+        GpuPreprocess,
```

Comment:
This feels a bit generic. Is that intentional so that the node can be used for multiple things? I don't know if that would be a good idea.

```suggestion
        PreprocessGpuInstanceData,
```

Reply:
This will eventually be for culling too. In fact, not doing culling here would be both difficult (requiring at least an extra buffer) and slow.